src/main/java/org/embulk/filter/join_file/FilteredPageOutput.java

```diff
@@ -94,6 +94,9 @@ else if (Types.LONG.equals(inputColumn.getType())) {
             else if (Types.TIMESTAMP.equals(inputColumn.getType())) {
                 pageBuilder.setTimestamp(inputColumn, pageReader.getTimestamp(inputColumn));
             }
+            else if (Types.JSON.equals(inputColumn.getType())) {
```

Comment:
Could you divide the p-r to Upgrading 0.8.x and Supporting JSON? This is because supporting JSON has more changes and discussions.